src/material/list/selection-list.ts

```diff
@@ -108,6 +109,7 @@ export class MatSelectionListChange {
     // be placed inside a parent that has one of the other colors with a higher specificity.
     '[class.mat-accent]': 'color !== "primary" && color !== "warn"',
     '[class.mat-warn]': 'color === "warn"',
+    '[class.mat-single-selected]': 'selected && !selectionList.multiple',
```

Comment:
We typically include the component name in the class: `.mat-list-single-item-selected` (except for the theme classes which are universal)

Reply:
Done